mobile/native/src/trade/position/handler.rs

```diff
@@ -206,3 +210,14 @@ pub async fn close_position() -> Result<()> {
     db::insert_order(order)?;
     update_position_after_dlc_closure(order)
 }
+
+/// Get the current position
+///
+/// FIXME: We are assuming that there is only one position open. This will
+/// change when we support multiple trading pairs.
+pub fn get_current_position() -> Result<Position> {
+    Ok(db::get_positions()?
```

Comment:
🔧 It is very likely that we don't have position. e.g. First startup. I suggest we return an Option `None` if no position was found, instead of failing.

Reply:
this is just a refactor (it was exactly like this in the code already), and I had to stop somewhere. follow-up PRs welcome.

Reply:
well isn't this now behaving differently? I might be reading it wrong, but wouldn't this logic now return an error if the position is not found?

Before your refactoring we were doing something like that.

```
if let Some(position) = db::get_positions()?.first() {
```